src/ONYXKEYS.js

```diff
@@ -180,8 +180,8 @@ export default {
     // Is Keyboard shortcuts modal open?
     IS_SHORTCUTS_MODAL_OPEN: 'isShortcutsModalOpen',
 
-    // Is close acount modal open?
-    IS_CLOSE_ACCOUNT_MODAL_OPEN: 'isCloseAccountModalOpen',
+    // Data from when user attempts to close their account (loading status and error message)
+    CLOSE_ACCOUNT_DATA: 'closeAccountData',
```

Comment:
NAB, `data` is an unnecessary description here because all things in Onyx are "data". The rest of the code would be more concise if we drop it e.g. `closeAccount.isLoading`, `closeAccount.error`, etc.

Reply:
Done, good call